### PR TITLE
Fixes the broken maven link in ubuntu build document

### PR DESCRIPTION
**Pull request by laneser**
We are excited to review your PR.

So we can do the best job, please check:

- [x] There's a descriptive title that will make sense to other developers some time from now. 
- [ ]  There's associated issues. All PR's should have issue(s) associated - unless a trivial self-evident change such as fixing a typo. You can use the format `Fixes #nnnn` in your description to cause GitHub to automatically close the issue(s) when your PR is merged.
- [x] Your change description explains what the change does, why you chose your approach, and anything else that reviewers should know.
- [ ] You have included any necessary tests in the same PR.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/spark?pullRequest=556) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/spark?pullRequest=556) before we can accept your contribution.<br/><br/>:x: laneser [sign now](https://cla.dotnetfoundation.org/dotnet/spark?pullRequest=556)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/spark?pullRequest=556) it.</sub>

File: docs/building/ubuntu-instructions.md [hunk 1]
```diff
@@ -54,11 +54,11 @@ If you already have all the pre-requisites, skip to the [build](ubuntu-instructi
...
-       Apache Maven 3.6.0 (97c98ec64a1fdfee7767ce5ffb20918da4f719f3; 2018-10-24T18:41:47Z)
-       Maven home: ~/bin/apache-maven-3.6.0
-       Java version: 1.8.0_191, vendor: Oracle Corporation, runtime: /usr/lib/jvm/java-8-openjdk-amd64/jre
-       Default locale: en, platform encoding: UTF-8
-       OS name: "linux", version: "4.4.0-17763-microsoft", arch: "amd64", family: "unix"
+       Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
+       Maven home: ~/bin/apache-maven-3.6.3
+       Java version: 1.8.0_242, vendor: Oracle Corporation, runtime: /usr/lib/jvm/java-8-openjdk-amd64/jre
+       Default locale: en_US, platform encoding: ANSI_X3.4-1968
+       OS name: "linux", version: "4.4.0-142-generic", arch: "amd64", family: "unix"
```
Comment (suhsteve) [inline docs/building/ubuntu-instructions.md:61]: nit: These are dependent on the platform you're running the mvn command on.

LGTM.

Comment (suhsteve) [reply docs/building/ubuntu-instructions.md:61]: You have updated the PR with files that aren't related with the original intentions of fixing the maven link instructions.  Can you create a separate PR with the new files?

**Comment by laneser** [comment]
Sorry, because I am not familiar with the github merge op.
I should close/cancel the PR.

File: docs/building/ubuntu-instructions.md (hunk 1 above)
Comment (laneser) [reply docs/building/ubuntu-instructions.md:61]: I created a seperate PR, thanks for your notification.

**Pull request closed without merging**